core/src/main/scala/algebra/Order.scala

```diff
@@ -148,6 +163,27 @@ object Order extends OrderFunctions {
    */
   implicit def ordering[A](implicit ev: Order[A]): Ordering[A] =
     new Ordering[A] {
-      def compare(x: A, y: A) = ev.compare(x, y)
+      def compare(x: A, y: A): Int = ev.compare(x, y)
+    }
+
+  /**
+   * A `Monoid[Order[A]]` can be generated for all `A` with the following
+   * properties:
+   *
+   * `empty` returns a trivial `Order[A]` which considers all `A` instances to
+   * be equal.
+   *
+   * `combine(x: Order[A], y: Order[A])` creates an `Order[A]` that first
+   * orders by `x` and then (if two elements are equal) falls back to `y`.
+   *
+   * @see [[Order.andThen]]
+   */
+  implicit def orderMonoid[A]: Monoid[Order[A]] =
```

Comment:
I don't think I'm too attached to this being implicit as long as there is a method for combining `Order` instances in this manner on the `Order` trait.